src/pages/gd/Swap/SwapCelo/UniSwap.tsx

```diff
@@ -97,7 +110,7 @@ export const UniSwap = (): JSX.Element => {
     }, [connect])
 
     const handleError = useCallback(async (e) => {
-        console.log('handleError -->', { e })
+        sendData({ event: 'swap', action: 'swap_failed', error: e })
```

Comment:
you cant send the error object to analytics. it has to be e.message
make sure it is like so in other places

Reply:
the format `event: X, action:Y` is for the old google analytics. in ua4 action is not required.
we should send events like `swap_failed`

Reply:
okay, I'll discuss with @drptiago

Reply:
the event format shouldnt be a blocker to release this. but the error need to be fixed

Reply:
okay done